suseconnect/usage.txt

```diff
@@ -42,6 +42,7 @@ Manage subscriptions at https://scc.suse.com
                              services installed by SUSEConnect.
         --rollback           Revert the registration state in case of a failed
                              migration.
+        --version            Print program version.
```

Comment:
In the Ruby SUSEConnect,  `--version` is between `--root` and `--debug` - but I'm not sure it really is a `Common option`

Reply:
Yes, as implemented it is not a common option but more a command, where only one of them will be executed. The ruby version also calls exit directly before making use of any other command:
https://github.com/SUSE/connect/blob/f874c2d0b801402f34b8d60eb75fbef40937603e/lib/suse/connect/cli.rb#L210